week4/week4-project/src/components/skeleton/LoadingSection.tsx

```diff
@@ -0,0 +1,39 @@
+import styled from "styled-components";
+import WeatherCardSkeleton from "./WeatherCardSkeleton";
+
+export interface LoadingSectionProps {
+  isDaily: boolean;
+}
+
+const LoadingSection = (props: LoadingSectionProps) => {
+  const { isDaily } = props;
```

Comment:
나나 궁금한거

```suggestion
const LoadingSection = ({ isDaily }: LoadingSectionProps) => {
```

이렇게 적지 않는 이유는 무엇인가요?????